includes/definitions/trendnet.yaml

```diff
@@ -0,0 +1,12 @@
+os: trendnet
+text: 'TRENDnet Switch'
+type: network
+icon: trendnet
+nobulk: true
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+      #mib_dir:
+      #    - trendnet
```

Comment:
Please add the mib file (best) or remove this comment.

Reply:
Removed it, thanks!